src/absinthe_parser.yrl

```diff
@@ -263,7 +263,24 @@ extract_atom({Value, _Line}) -> Value.
 extract_binary(Value) when is_binary(Value) -> Value;
 extract_binary({Token, _Line}) -> list_to_binary(atom_to_list(Token));
 extract_binary({_Token, _Line, Value}) -> list_to_binary(Value).
-extract_quoted_string_token({_Token, _Line, Value}) -> list_to_binary(lists:sublist(Value, 2, length(Value) - 2)).
+extract_quoted_string_token({_Token, _Line, Value}) -> iolist_to_binary(unescape(lists:sublist(Value, 2, length(Value) - 2))).
+
+unescape(Escaped) -> unescape(Escaped, []).
+
+unescape([], Acc) -> lists:reverse(Acc);
+unescape([$\\, $" | T], Acc) -> unescape(T, [16#0022 | Acc]);
```

Comment:
Can do. I just wanted to show the replacements conform to the spec, but I guess the character literals will be more robust and easily understandable